
Header header

string head
float64 timeUTC

string status

float64 latitude
string northOrSouth
float64 longitude
string eastOrWest

float64 velocity
float64 yaw
string dayUTC
float64 magneticYaw
string magneticEorW
string mode
